src/api/vdom.js

```diff
@@ -89,7 +89,7 @@ const attributesContext = propContext(attributes, {
 
   // Default attribute applicator.
   [symbols.default] (elem, name, value) {
-    const { props, prototype } = customElements.get(elem.tagName) || {
+    const { props, prototype } = customElements.get(elem.localName) || {
```

Comment:
should we be normalising it here then? Or is `customElements.get` not case sensitive?